src/styles/blocks/main.scss

```diff
@@ -0,0 +1,47 @@
+@import "./../utils/variables";
+
```

Comment:
the main.scss file is to @import all stiles together
you shold move .calendar styles to separate file ./styles/blocks/calendar.scss
the file main.scss should be in folder /styles/main.scss
files normalize.scss and reset.scss shold be in folder ./styles/
folder 'utils' is for variables, functions, mixins

Reply:
i think, name file is not importent, i use style.scss to connect all styles